sonic-psud/scripts/psud

```diff
@@ -272,13 +272,13 @@ class PsuChassisInfo(logger.Logger):
 
 
 class PsuStatus(object):
-    def __init__(self, logger, psu):
-
+    def __init__(self, logger, psu, psu_index):
         self.psu = psu
-        self.presence = False
-        self.power_good = False
-        self.voltage_good = False
-        self.temperature_good = False
+        self.psu_index = psu_index
+        self.presence = True
```

Comment:
why do we need to change the variable from False to True here?.

Reply:
If it is False, and the PSU is in a bad state, for example, PSU voltage is not good, there will be no error logs.